compiler/options.nim

```diff
@@ -213,7 +213,7 @@ type
     strictFuncs,
     views,
     strictNotNil,
-    overloadableEnums,
+    overloadableEnums, # not experimental anymore
```

Comment:
Why the comment?

Reply:
Other features above have the same comment, if it's removed from the enum then `{.experimental: "overloadableEnums".}` becomes an error